CONTRIBUTING.md

```diff
@@ -34,6 +34,59 @@ We'll try to be as responsive as possible in reviewing and accepting pull reques
 - If `index.yaml` gets out of date on the repository, see
   [`./scripts/`](./scripts) for a workflow to fix
 
+## Templates
+
+The `rstudio-workbench` and `rstudio-connect` charts both make heavy use of the "templating" feature of the Posit Job
+Launcher. The templates are embedded within the charts using the following paradigm:
+
+- copy "parent" templates as-is into the [./examples/launcher-templates/default](./examples/launcher-templates/default)
```

Comment:
Maybe note that the template version is not the launcher version.

Reply:
Don't love my wording... but I gave it a shot 😅